onedal/cluster/kmeans.py

```diff
@@ -139,15 +139,18 @@ def _check_params_vs_input(
     def _get_policy(self, queue, *data):
         return _get_policy(queue, *data)
 
-    def _get_onedal_params(self, dtype=np.float32):
+    def _get_onedal_params(self, dtype=np.float32, result_options=None):
```

Comment:
```suggestion
    def _get_onedal_params(self, dtype=np.float32, result_options=""):
```
Could remove below if statement and set default to empty string to simplify a little